### PR TITLE
feat(breadcrumbItem, table): removed breadcrumbItem and data table ellipsis prop

### PR DESCRIPTION
**Pull request by tulasigudibanda**
## Summary

Brief summary describing the changes here.

## ADO Story or GitHub Issue Link

[TASK 1931296](https://dev.azure.com/Kyndryl/Shidoka%20-%20Bridge%20Design%20System/_workitems/edit/1931296)
[TASK 1931290](https://dev.azure.com/Kyndryl/Shidoka%20-%20Bridge%20Design%20System/_workitems/edit/1931290)

## Notes

Removed deprecated breadcrumbItem component 
Removed data table ellipsis prop

## Checklist

- [ ] Used Conventional Commit messages as outlined in the contributing guide.
  - [ ] Noted breaking changes (if any).
- [ ] Documented/updated all props, events, slots, parts, etc with JSDoc.
  - [ ] Ran the `analyze` command to update Storybook docs.
- [ ] Added/updated Stories with controls to cover all variants.
- [ ] Ran `test` locally to address any failures.
- [ ] Added/updated the Figma link for the Story's Design tab.
- [ ] Added any new component exports to the src/index.ts file

### CODE REVIEW

File: src/stories/migration.mdx [hunk 1]
```diff
@@ -8,6 +8,8 @@ Breaking Changes
 
 ## 2.0
 
+- Removed BreadcrumbItem component
+- Removed Data table ellipsis prop
```
Comment (brian-patrick-3) [inline src/stories/migration.mdx:12]: Put these notes under their respective category headings below

Comment (tulasigudibanda) [reply src/stories/migration.mdx]: Done

**Review by brian-patrick-3 — APPROVED** [review]

**Review by kumarivijeta — APPROVED** [review]

**Review by jaideep08 — APPROVED** [review]

**Pull request merged**